src/main/java/seedu/address/model/StateManager.java

```diff
@@ -89,4 +113,20 @@ public void redo() throws CommandException, IllegalValueException {
             currentCommand.executeCommand();
         }
     }
+
+    /**
+     * Restores previous data (i.e undo a clear command )
+     *
+     * @throws CommandException
+     * @throws IllegalValueException
+     */
+    public void restoreData() throws CommandException, IllegalValueException {
+        if (previousDataStack.isEmpty()) {
+            // Can't undo as no history
```

Comment:
Do you think this comment is redundant? It seems quite obvious what the condition is checking.